### PR TITLE
Update rector-migrate.php to support latest Rector

### PR DESCRIPTION
**Pull request by alexandreelise**
Support Rector 0.13.x features mainly RectorConfig

### CODE REVIEW

**Comment by alexandreelise** [comment]
Hello codingmachine team I watched an old PHP talk from 2018 last night they talked about your project safe. I gave it a try following the instructions in your README but the composer require --dev rector/rector installed the latest version of rector which doesn't use the ContainerConfigurator anymore but rather their own RectorConfig. Here is the author official blog post about this change: [related blog post of Rector author](https://getrector.org/blog/new-in-rector-012-introducing-rector-config-with-autocomplete)

Thanks for your awesome work have a great day.

**Comment by dbrekelmans** [comment]
Hey @alexandreelise, looks good to me! If you could fix the failing checks, then everything will be good to merge 👍

**Comment by alexandreelise** [comment]
> Hey @alexandreelise, looks good to me! If you could fix the failing checks, then everything will be good to merge +1

Hey @dbrekelmans I'll look into it thanks for the feedback. Happy to help as much as I can.

**Comment by alexandreelise** [comment]
Hey @dbrekelmans I just pushed the changes as you requested can you give a check please?

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 vendor/
 /generator/tests/rector/composer.lock
 .phpunit.result.cache
+.idea
```
Comment (dbrekelmans) [inline .gitignore:9]: This is user-specific and should not be in this repo. You could add this to your local global gitignore file https://gist.github.com/subfuzion/db7f57fff2fb6998a16c
```suggestion
```

**Review by dbrekelmans — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (alexandreelise) [reply .gitignore:9]: ok

Comment (alexandreelise) [reply .gitignore]: Hi @dbrekelmans your suggested change was solved in https://github.com/thecodingmachine/safe/pull/372/commits/8e6f7c54be36d87451e19d0f6651a6a1936db273

**Comment by alexandreelise** [comment]
I completely forgot to read the CONTIBUTING guideline. My bad. Now that I red it I followed the steps it should work as expected. Please review @dbrekelmans . Have a great day.

**Review by dbrekelmans — APPROVED** [review]

**Comment by alexandreelise** [comment]
Happy to contribute. Have a great day @dbrekelmans and thecodingmachine team. Take care.

**Pull request merged**